backend.tf

```diff
@@ -0,0 +1,25 @@
+# Storing terraform state in the cloud
```

Comment:
I don't think we support locking on the S3 backend. It would be more interesting to use a pg backend.